ui/lib/core/addon/components/info-table-item-array.js

```diff
@@ -62,29 +63,26 @@ export default class InfoTableItemArray extends Component {
     return displayArray;
   }
 
-  async checkWildcardInArray() {
-    if (!this.args.displayArray) {
-      return;
-    }
-    let filteredArray = await this.args.displayArray.filter((item) => isWildcardString([item]));
-    this.wildcardInDisplayArray = filteredArray.length > 0 ? true : false;
-  }
-
-  @task *fetchOptions() {
+  @action async fetchOptions() {
     if (this.args.isLink && this.args.modelType) {
-      let queryOptions = {};
+      let queryOptions = this.args.backend ? { backend: this.args.backend } : {};
 
-      if (this.args.backend) {
-        queryOptions = { backend: this.args.backend };
-      }
+      let modelRecords = await this.store.query(this.args.modelType, queryOptions).catch((err) => {
+        if (err.httpStatus === 404) {
+          return [];
+        } else {
+          return null;
+        }
+      });
 
-      let options = yield this.store.query(this.args.modelType, queryOptions);
-      this.formatOptions(options);
+      this.allOptions = modelRecords ? modelRecords.mapBy('id') : null;
+      if (this.args.renderItemName && modelRecords) {
```

Comment:
since we're already querying in this component - decided to leverage this to render the name if the model is hydrated (as is the case with both `identity/entity` and `identity/group`)